1

```diff
@@ -0,0 +1,379 @@
+info: ghostty version=1.1.1-HEAD+3470f33c
```

Comment:
A log file that didn't get deleted perhaps?

Reply:
I'd approve the PR but it looks like it's failing to compile again, both in CI and for me locally.